examples/usb_serial.rs

```diff
@@ -0,0 +1,66 @@
+//! CDC-ACM serial port example using polling in a busy loop.
+#![no_std]
+#![no_main]
+
+extern crate panic_semihosting;
+
+use cortex_m_rt::entry;
+use stm32l0xx_hal::usb::{Peripheral, UsbBus};
+use stm32l0xx_hal::{pac, prelude::*, rcc};
```

Comment:
```suggestion
use stm32l0xx_hal::{pac, prelude::*, rcc, syscfg::SYSCFG};
```
Required below. See other comment.